src/toast/scripts/benchmarking_utilities.py

```diff
@@ -664,7 +724,14 @@ def create_input_maps(
         new=True,
         fwhm=np.radians(3.0 / 60.0),
     )
-    healpy.write_map(input_map_path, maps, nest=True, fits_IDL=False, dtype=np.float32)
+    healpy.write_map(
```

Comment:
If this is taking a long time, we might consider using the HDF5 format used in DC0 sims.

Reply:
The call the synfast is the slow part I believe.